apisix/plugins/file-logger.lua

```diff
@@ -59,6 +59,9 @@ local _M = {
     metadata_schema = metadata_schema
 }
 
+core.ctx.register_var("resp_body", function(ctx)
```

Comment:
Would be better to put it in https://github.com/apache/apisix/blob/master/apisix/core/ctx.lua so other loggers can use it too.

Reply:
I don't think this var should be put into the `ctx.lua` file. Because the value of resp_body comes from `ctx.resp_body`, which is set by `file-logger` when include_resp_body = true.

Reply:
Not only one logger plugin will support recording the response body. `file-logger.lua` is not an appropriate place.